lib/contracts/index.ts

```diff
@@ -44,11 +44,10 @@ const cards = [
 	userSettings,
 ];
 
-export const CARDS = cards.reduce<{ [slug: string]: ContractDefinition }>(
-	(acc, contract) => {
-		const initializedContract = initialize(contract as any);
-		acc[initializedContract.slug] = initializedContract;
-		return acc;
-	},
-	{},
-);
+export const CARDS = contracts.reduce<{
+	[slug: string]: ContractDefinition;
+}>((acc, kontract) => {
```

Comment:
Is this renaming necessary?

Reply:
I would say, we want to, because the point of this refactoring at all is to bring consistency in the language we use.

Reply:
@taromurao I mean `kontract`. I don't see any other changes in this file that could conflict with a plain `contract`.

Reply:
This is just to bypass a shadowed constant linting error.